src/seedu/addressbook/Main.java

```diff
@@ -80,6 +80,16 @@ private void exit() {
     private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
+            // check whether the storage file still exist
+        	try {
+            	this.storage.checkFileExistence();
+         		} catch (MissingFileException error) {
+         				ui.showToUser(error.getMessage());
+         				
+         				//if the file is missing just simply terminate the program
+         				exit();
+         		}
+
```

Comment:
Think about what the difference is between your codes here and the belowing
```
if (!this.storage.checkFileExistence()) {
  ui.showToUser(error.getMessage());
  exit();
}
```
Is exception really used in this way?